src/main/java/gtPlusPlus/xmod/gregtech/common/tileentities/machines/multi/production/turbines/GT_MTE_LargeTurbine_SCSteam.java

```diff
@@ -55,14 +55,15 @@ public int getFuelValue(FluidStack aLiquid) {
     }
 
     @Override
-    int fluidIntoPower(ArrayList<FluidStack> aFluids, long aOptFlow, int aBaseEff, float[] flowMultipliers) {
+    long fluidIntoPower(ArrayList<FluidStack> aFluids, long aOptFlow, int aBaseEff, float[] flowMultipliers) {
         int tEU = 0;
         int totalFlow = 0; // Byproducts are based on actual flow
         int flow = 0;
-        int remainingFlow = MathUtils.safeInt((long) (aOptFlow * 1.25f)); // Allowed to use up to 125% of optimal flow.
-                                                                          // Variable required outside of loop for
+        // Variable required outside of loop for
         // multi-hatch scenarios.
         this.realOptFlow = (double) aOptFlow * (double) flowMultipliers[0];
```

Comment:
I think this was just here from a copy paste error or something like that.
SC does not (!) use flowmultipliers. see https://github.com/GTNewHorizons/GoodGenerator/blob/master/src/main/java/goodgenerator/blocks/tileEntity/SupercriticalFluidTurbine.java

Reply:
Well, I don't know what original author thought, maybe we can fix that together.

Reply:
And if necessery we can add a new multiplier to SC steam for turbines.

Reply:
we can talk with some more devs. But I dont think we want to buff SC steam. its already the meta for plasma.

Reply:
done.

Reply:
thanks! a material specific multiplier for SC will have to wait I think until someone really wants go into balancing SC as a whole. (steelux agreed with me on discord)
I made a PR to add SC to the tooltip so it is clear to the player.

Reply:
Tooltip is really misleading, I wasted a whole day build fusions for duranium :(

Reply:
https://github.com/GTNewHorizons/GT5-Unofficial/pull/2515